mysql-test/suite/galera/r/galera_alter_table_algorithm.result

```diff
@@ -0,0 +1,64 @@
+connection node_2;
+connection node_1;
+CREATE TABLE t1 (f1 INTEGER) ENGINE=InnoDB;
+set global wsrep_on=OFF;
+reset master;
+set global wsrep_on=ON;
+connection node_2;
+set global wsrep_on=OFF;
+reset master;
+set global wsrep_on=ON;
+connection node_1;
+ALTER TABLE t1 ADD COLUMN f2 INTEGER, ALGORITHM=INSTANT;
+show binlog events limit 1 offset 4;
+Log_name	Pos	Event_type	Server_id	End_log_pos	Info
+binlog.000001	367	Query	1	489	use `test`; ALTER TABLE t1 ADD COLUMN f2 INTEGER, ALGORITHM=INSTANT
```

Comment:
Many of the fields here (`Pos`, `End_log_pos`, and maybe `Log_name` and `Server_id`?) will vary from run to run of this test.

The only thing the test result actually cares about (I think) is the `Info` column, right?

Consider adding this before each invocation of `show binlog events …` in the `.test` file:

```
--replace_column 1 LOG_NAME 2 POS 4 SERVER_ID 5 END_LOG_POS
```

See [mysql-test/main/alter_table.test](https://github.com/MariaDB/server/blob/11.0/mysql-test/main/alter_table.test#L252-L261), for example, where this is used in other pre-existing tests.